README.md

```diff
@@ -45,6 +45,14 @@ Use them to instead of `trace()`, `Sys.exit()`, etc, for maximum compatibility a
 - `travix.Logger.println(string)`: Print a string with newline
 - `travix.Logger.exit(exitCode)`: Exit the process
 
+If you want to be able to run your tests e.g. locally without Travix dependency use conditional compilation like this:
+```haxe
+#if travix
```

Comment:
Probably only relevant for haxelibs. I added Travis/travix support for my [https://github.com/vegardit/haxe-doctest] library but don't want to add a hard but optional dependency on travix.